### PR TITLE
Fixes flightsuit being able to teleport through welded airlocks

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Comment by ma44** [comment]
Feature

**Comment by coiax** [comment]
Make the changelog more descriptive please, otherwise everyone keeps asking in ooc

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 1]
```diff
@@ -541,6 +541,7 @@
 		pass += A.locked
 		pass += A.stat	//No power, no automatic open
 		pass += A.emagged
+		pass += A.welded
```
Comment (KorPhaeron) [inline code/modules/clothing/spacesuits/flightsuit.dm]: Isn't there a proc to check if it can open already?

**Comment by Kromgar** [comment]
It flies over the airlock clearly

**Comment by kevinz000** [comment]
Not like we have roofs.

File: code/modules/clothing/spacesuits/flightsuit.dm (hunk 1 above)
Comment (AnturK) [inline code/modules/clothing/spacesuits/flightsuit.dm]: Turns out there isn't, can be added later.

**Pull request merged**